nlds/rabbit_connection.py

```diff
@@ -1,20 +1,23 @@
-from configparser import ConfigParser
```

Comment:
Please add the license headers to each file.
I realise you've inherited files with license headers from me! :)